src/legacy/core_plugins/data/public/query/query_bar/components/query_bar_input.tsx

```diff
@@ -498,28 +509,23 @@ export class QueryBarInputUI extends Component<Props, State> {
                 }}
                 autoComplete="off"
                 spellCheck={false}
-                aria-label={
-                  this.props.screenTitle
-                    ? this.props.intl.formatMessage(
-                        {
-                          id: 'data.query.queryBar.searchInputAriaLabel',
-                          defaultMessage:
-                            'You are on search box of {previouslyTranslatedPageTitle} page. Start typing to search and filter the {pageType}',
-                        },
-                        {
-                          previouslyTranslatedPageTitle: this.props.screenTitle,
-                          pageType: this.props.appName,
-                        }
-                      )
-                    : undefined
-                }
+                aria-label={this.props.intl.formatMessage(
+                  {
+                    id: 'data.query.queryBar.searchInputAriaLabel',
+                    defaultMessage: 'Start typing to search and filter the {pageType} page',
+                  },
+                  {
+                    pageType: this.props.appName,
+                  }
+                )}
                 type="text"
                 data-test-subj="queryInput"
                 aria-autocomplete="list"
-                aria-controls="kbnTypeahead__items"
-                aria-activedescendant={
-                  this.state.isSuggestionsVisible ? 'suggestion-' + this.state.index : ''
-                }
+                {...(this.state.isSuggestionsVisible && { 'aria-controls': 'kbnTypeahead__items' })}
```

Comment:
Code Style: Setting a property to `undefined` will also not render it in HTML, so we usually try to use the following version:

```js
aria-controls={this.state.isSuggestionsVisible ? 'kbnTypeahead__items' : undefined }
```

since that reads a bit nicer (and lints better, as you figured out above :D) then the spread operator.